modules/integration/tests-integration/tests-mediator-2/src/test/resources/artifacts/ESB/defaultconfigs/synapse.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+<!--
+  ~  Copyright (c) 2005-2015, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+  ~
+  ~  WSO2 Inc. licenses this file to you under the Apache License,
+  ~  Version 2.0 (the "License"); you may not use this file except
+  ~  in compliance with the License.
+  ~  You may obtain a copy of the License at
+  ~
+  ~    http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~  Unless required by applicable law or agreed to in writing,
+  ~  software distributed under the License is distributed on an
+  ~  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~  KIND, either express or implied.  See the License for the
+  ~  specific language governing permissions and limitations
+  ~  under the License.
+  -->
+
+<!-- An empty flat synapse configuration shipped with the WSO2 ESB -->
+<definitions xmlns="http://ws.apache.org/ns/synapse">
+
+    <!-- You can add any flat sequences, endpoints, etc.. to this synapse.xml file if you do
+    *not* want to keep the artifacts in several files -->
+    <taskManager provider="org.wso2.carbon.mediation.ntask.NTaskTaskManager"/>
+</definitions>
```

Comment:
@chanakaudaya the idea was to make to make sure to follow the same pattern that is being used now. For any test, configs are taken from from the /**MODULE**/src/test/resources/artifacts/ESB folder, which is inside the module folder. Also it is directly used when applying the configurations for each test using TestConfigurationProvider.getResourceLocation("ESB") statement. @jagathsisira  and I have discussed the matter and decided to keep the same way followed at present to avoid confusion and retain same structure.